pkg/cli/admin/catalog/mirror.go

```diff
@@ -48,7 +48,12 @@ var (
 
 		A mapping.txt file is also created that is compatible with "oc image mirror". This may be used to further
 		customize the mirroring configuration, but should not be needed in normal circumstances.
-	`)
+
+		Mirroring into an airgapped environment requires two steps: mirroring from the catalog to local files,
+		and then again from local files into the airgapped registry. See examples ofr the "--remap" flag for more
```

Comment:
ofr/of 
also, I don't see any examples of using --remap flag, you will add those?